about.html

```diff
@@ -0,0 +1,126 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link href="http://fonts.cdnfonts.com/css/cocogoose" rel="stylesheet" />
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css" />
+<link href="https://fonts.googleapis.com/css?family=Lato:400,600,500,700,800" rel="stylesheet">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <link rel="stylesheet" href="./styles/about.css">
+    <title>VueJs | Amsterdam</title>
+</head>
+<body>
+    <nav class="nav-header">
+        <i class="fa fa-facebook nav-icons" aria-hidden="true"></i>
+        <i class="fa fa-twitter nav-icons" aria-hidden="true"></i>
+        <ul class="nav-header-list">
+            <li>English</li>
+            <li>My Page</li>
+            <li>Logout</li>
+        </ul>
+    </nav>
```

Comment:
- [ ] OPTIONAL: I have noticed that when mobile `nav` is open, users can still scroll the page, and at some point content of the page gets on top of `navbar`.  I strongly recommend you to add overflow:hidden to body in order to prevent this behavior: 
![image](https://user-images.githubusercontent.com/77038610/152559454-90e54418-02e9-4995-8411-aa46aedbfcc4.png)

Reply:
This will also be included in the issue